src/migrations/options.rs

```diff
@@ -119,6 +119,10 @@ pub struct Migrate {
     /// a regular `.edgeql` file, after which the above query will return nothing.
     #[arg(long)]
     pub dev_mode: bool,
+
+    /// Runs the migration(s) in a single transaction (legacy behaviour).
```

Comment:
Wanting a single transaction might be a legitimate case, so lets not scare users with "legacy".

```suggestion
    /// Runs the migration(s) in a single transaction.
```